### ISSUE TITLE
Bing, Systran, and Baidu doesn't work with the latest version

### ISSUE THREAD

**Issue by DemonKiller1991** [Sinflower/Translation-Aggregator#20]
Hi Sinflower

Was wondering if you could look into the Bing, Systran and Baidu issues with the latest release. They're not working. I've also faced the same issues with the previous releases, so most likely they changed it from their end.

Thank you in advance if you're already working on it.

**Comment by Sinflower** [comment]
Hey,
I did a few tests and could not detect any problems with Bing.
Systran and Baidu indeed stopped working, as both have implemented new advanced security features.
I will try to work around them, but from what I have seen so far, you better not get your hopes up for a fix any time soon.

**Comment by DemonKiller1991** [comment]
![image](https://user-images.githubusercontent.com/48101459/175872370-11fb91a6-a522-4182-9690-38f90fd54749.png)

That's the error message I receive whenever I use Bing.

**Comment by Sinflower** [comment]
![image](https://user-images.githubusercontent.com/23295734/175975439-5b57fe20-3bd9-4beb-baf8-72d6a366ea2a.png)
This is how it looks for me using the latest release.
I also looked at the Bing web request and from what I can see, no changes have been made.
You can try this build, it uses a newer compiler, maybe that helps.
[Translation Aggregator.zip](https://github.com/Sinflower/Translation-Aggregator/files/8993247/Translation.Aggregator.zip)

**Comment by DemonKiller1991** [comment]
![image](https://user-images.githubusercontent.com/48101459/175987565-ebf956e7-f050-46b0-8de9-78ecc0e0f9fb.png)
Still shows the same error

**Comment by Sinflower** [comment]
Sooooo I finally managed to reproduce the error, your final screenshot helped 👍.
I did not know that you were using Chinese text, I always tested with Japanese text.
Using Chinese text I got the same error as you. Turns out that the Chinese language names were changed at one point.
Try this build and see if it finally works.
[Translation Aggregator.zip](https://github.com/Sinflower/Translation-Aggregator/files/8994538/Translation.Aggregator.zip)
